src/Microsoft.Extensions.Hosting.Abstractions/Microsoft.Extensions.Hosting.Abstractions.csproj

```diff
@@ -8,6 +8,7 @@
     <NoWarn>$(NoWarn);CS1591</NoWarn>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <PackageTags>hosting</PackageTags>
+    <EnableApiCheck>false</EnableApiCheck>
```

Comment:
We will generate a baseline file for this when we RTM. This is a new package in 2.0.